src/v2/view-builder/internals/BaseModel.js

```diff
@@ -80,6 +87,22 @@ const create = function (remediation = {}, optionUiSchemaConfig = {}) {
   const BaseModel = Model.extend({
     props,
     local,
+    toJSON: function () {
+      const modelJSON = Model.prototype.toJSON.apply(this, arguments);
+
+      const identifier = this.get('identifier');
+      if (identifier) {
```

Comment:
shall it be `identifier && currentFormName is one of Object.keys(FORM_NAME_TO_OPERATION_MAP)`?

Reply:
That would make this setting bound to our upkeep of this Enum, which would be annoying for us and users.